Gems/Atom/Feature/Common/Code/Source/RayTracing/RayTracingPass.h

```diff
@@ -57,15 +61,24 @@ namespace AZ
             // load the raytracing shaders and setup pipeline states
             void CreatePipelineState();
 
-            // helper for loading a shader from a shader asset reference
-            Data::Instance<RPI::Shader> LoadShader(const RPI::AssetReference& shaderAssetReference);
-
             // pass data
             RPI::PassDescriptor m_passDescriptor;
             const RayTracingPassData* m_passData = nullptr;
 
+            Name m_fullscreenSizeSourceSlot;
+            bool m_fullscreenDispatch = false;
+            RPI::PassAttachmentBinding* m_fullscreenSizeSourceBinding = nullptr;
+
+            bool m_indirectDispatch = false;
+            Name m_indirectDispatchBufferSlot;
```

Comment:
Can you change the variable name from m_indirectDispatchBufferSlot to m_indirectDispatchBufferSlotName to make it explicit that it's the name and not the slot?
This would help with code readability.

Reply:
done